app/controllers/resources_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class ResourcesController < AdminController
+  skip_after_action :verify_authorized, only: [:index]
+  skip_after_action :verify_policy_scoped, only: [:index]
```

Comment:
I have deliberately skipped these actions instead of adding a Resources policy which would always return true. The page is still protected behind authentication. If the rules to access resources grow more complex, we can add a policy.